# Export the default variables if they aren't set.
# Otherwise we end up with inconsistently defined/exported
# defaults.
# Solaris-specific patch.
#
# I don't fully understand what this does. It would be worth discussing with
# upstream. They clerly were using the commands at some point in time.

--- variables.c	2011-01-24 17:07:48.000000000 -0800
+++ variables.c	2011-11-29 09:29:51.902791524 -0800
@@ -471,14 +471,12 @@
   /* Now make our own defaults in case the vars that we think are
      important are missing. */
   temp_var = set_if_not ("PATH", DEFAULT_PATH_VALUE);
-#if 0
+
   set_auto_export (temp_var);	/* XXX */
-#endif
 
   temp_var = set_if_not ("TERM", "dumb");
-#if 0
+
   set_auto_export (temp_var);	/* XXX */
-#endif
 
 #if defined (__QNX__)
   /* set node id -- don't import it from the environment */
